Gopkg.toml

```diff
@@ -136,9 +136,9 @@ required = [
   name = "k8s.io/apiextensions-apiserver"
   version = "kubernetes-1.13.2"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.13.1"
```

Comment:
I couldn't make it work other than with the override and going back one patch version. That is some weird upstream thing with the currently available version.

Reply:
This is something that `cluster-api` dependency brought?

Reply:
Yes.